knowledge-base/generate-excel-files-from-ienumerable-collections.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Generating Excel Documents from IEnumerable Collections
+description: Learn how to create Excel files from IEnumerable Collections using the RadSpreadProcessing library.
+type: how-to
+page_title: How to  Excel Documents from IEnumerable Collections 
+slug: generate-excel-files-from-ienumerable-collections
+tags: radspreadprocessing, document processing, excel, formatting, datatable, ienumerable
+res_type: kb
+category: knowledge-base
+ticketid: 1653503
+---
+
+## Environment
+
+| Version | Product | Author | 
+| --- | --- | ---- | 
+| 2024.2.426| RadSpreadProcessing |[Desislava Yordanova](https://www.telerik.com/blogs/author/desislava-yordanova)| 
+
+## Description
+
+This article demonstrates a sample approach how to generate Excel documents from IEnumerable collections.
+
+## Solution
+
+ RadSpreadProcessing is the perfect fit for the requirement. It is possible to iterate a collection of any custom objects and populate the worksheet's cells with the values coming from the respective fields in the custom objects. A good example of a sample approach how to do it is available in the following code snippet which fills the data in a worksheet from a collection of Employee objects.
```

Comment:
Space at the start of the sentence.

Reply:
A good example of [a sample approach](this is not needed) how to do it is available ...